howso/client/pandas/client.py

```diff
@@ -231,14 +227,14 @@ def react_series(
                     An aggregated list of any requested details.
         """
         trainee_id = self._resolve_trainee_id(trainee_id)
-        feature_attributes = self.trainee_cache.get(trainee_id).features
+        feature_attributes = self.trainee_cache.get(trainee_id)["features"]
         response = super().react_series(trainee_id, *args, series_index=series_index, **kwargs)
 
-        response['action'] = format_dataframe(response.get("action"), feature_attributes)
+        response['action'] = utilities.format_dataframe( response['get']("action"), feature_attributes)
```

Comment:
Incorrect accessor `['get']`
```suggestion
        response['action'] = utilities.format_dataframe(response.get("action"), feature_attributes)
```

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60

Reply:
Changes still missing in latest commit